index.html

```diff
@@ -934,6 +943,13 @@ <h3>Processing Model</h3>
 surface resource fetch events.</p>
 <div class="issue" data-number="27"></div>
 <ol data-link-for="PerformanceResourceTiming">
+<li>If the resource's <a data-cite="Fetch#concept-request">Request</a>'s
+<a data-cite="Fetch#concept-request-destination">destination</a> equals to
+"document", "object" or "embed", and was not triggered by 
```

Comment:
If the destination is "object" or "embed", then the request definitely comes from the attribute-processing algorithms.   Those are the only things that set  destination to "object" or "embed".   So we only need to special-case the "document" case here, I think.